### DISCUSSION TITLE
mes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum

### DISCUSSION THREAD

**Discussion by enzifiri** [enzifiri/GitHub-Achievements#279] [Q&A] [answered]
ook even slightly believable. If you

**Comment by mertcangokgoz** [comment, marked as answer]
It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
